src/DataContainer/DcaFilterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\DataContainer;
+
+/**
+ * DCA filter interface.
```

Comment:
We are currently using this for pickers only, but the `DataContainer::setDcaFilter()` method is meant to be a general method.

> @see DataContainer::setDcaFilter() or DC_Folder::setDcaFilter()

How about a short comment like above?